src/clientdynamicinfo.h

```diff
@@ -20,21 +20,39 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #pragma once
 
 #include "irrTypes.h"
+#include "settings.h"
+#include "client/renderingengine.h"
 
 
 struct ClientDynamicInfo
 {
+public:
 	v2u32 render_target_size;
 	f32 real_gui_scaling;
 	f32 real_hud_scaling;
 	v2f32 max_fs_size;
 
+	static const ClientDynamicInfo get_current() {
```

Comment:
This class is used on the server too. It shouldn't know about the rendering engine